tests/utils.py

```diff
@@ -7,3 +7,8 @@
 def load_fixture(fixture_file_name):
     with open(path.join(FIXTURES_DIR, fixture_file_name)) as fixture_file:
         return json.load(fixture_file)
+
+
+def read_fixture(fixture_file_name):
```

Comment:
[optional]: you could get `load_fixture` to simply `return json.loads(read_fixture(fixture_file_name))` for less code duplication

Reply:
fixed.